src/ck-core/ckrdmadevice.C

```diff
@@ -54,10 +54,23 @@
 #include "hapi.h"
 #include "gpumanager.h"
 
+#define TIMING_BREAKDOWN 1
+#if TIMING_BREAKDOWN
+#define N_TIMER 16
+#endif
+
 CsvExtern(GPUManager, gpu_manager);
 
 // Invoked after post entry method
 bool CkRdmaDeviceIssueRgets(envelope *env, int numops, void **arrPtrs, int *arrSizes, CkDeviceBufferPost *postStructs) {
+#if TIMING_BREAKDOWN
+  static double total_times[N_TIMER] = {0};
+  static int count = 0;
+  count++;
```

Comment:
Are these safe for SMP mode?

Reply:
Good point, I forgot that the timing breakdown was included in this patch. Will update it to use Cpv variables instead